gaas-ui/src/util/util.js

```diff
@@ -0,0 +1,8 @@
+export function isJSONString(str) {
+    try {
+        JSON.parse(str);
```

Comment:
This will return true for primitive types, you can change it to something like this to only accept objects: https://stackoverflow.com/a/20392392

Reply:
done